tests/lib/rules/jsx-curly-brace-presence.js

```diff
@@ -657,12 +657,6 @@ ruleTester.run('jsx-curly-brace-presence', rule, {
       options: [{ props: 'always' }],
       errors: [{ messageId: 'missingCurly' }],
     },
-    {
-      code: `<MyComponent prop="foo 'bar'">foo</MyComponent>`,
-      output: `<MyComponent prop={"foo 'bar'"}>foo</MyComponent>`,
-      options: [{ props: 'always' }],
-      errors: [{ messageId: 'missingCurly' }],
-    },
```

Comment:
Duplicate of:

https://github.com/jsx-eslint/eslint-plugin-react/blob/8e1a94b67d081fdc132e9a7e175db3fbf2e02956/tests/lib/rules/jsx-curly-brace-presence.js#L648-L653